com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 ### Added
 
 ### Changed
+- `unmanaged` structs are no longer universally accepted as RPC parameters because some structs (i.e., structs with pointers in them, such as `NativeList`) can't be supported by the universal struct serializer. Structs that are intended to be serialized across the network must add `ISerializeByMemcpy` to the interface list (i.e., `struct Foo : ISerializeByMemcpy`). This interface is empty and just serves to mark the struct as compatible with universal serialization. For external structs you can't edit, you can pass them to RPCs by wrapping them in `ForceSerializeByMemcpy<T>`. (#1901)
```

Comment:
doesn't this also mean that any existing code relying on a struct being serialized without the interface will immediately break after this change?

Reply:
Yep. But there's really no way to make `NativeList <T>` an error without making other things an error. We can choose to leave this part of it out, in which case we can fix template types causing crashes, but people will still be able to pass in NativeList and it will still crash... the alternative to doing this is documentation saying not to use NativeList in RPCs

Reply:
or we could add explicit support for NativeList and then that'll work, but NativeArray will crash... we can support all the things but there will always be something users can pass in without realizing it's invalid that will cause a crash.